Detectors/Base/include/DetectorsBase/MatLayerCylSet.h

```diff
@@ -118,6 +127,13 @@ class MatLayerCylSet : public o2::gpu::FlatObject
   static constexpr size_t getBufferAlignmentBytes() { return 8; }
 #endif // !GPUCA_GPUCODE
 
+  static constexpr float LayerRMax = 500;    // maximum value of R lookup (corresponds to last layer of MatLUT)
+  static constexpr float VoxelRDelta = 0.05; // voxel spacing for layer lookup; seems a natural choice - corresponding ~ to smallest spacing
+  static constexpr float InvVoxelRDelta = 1.f / VoxelRDelta;
+
+  std::vector<std::pair<uint16_t, uint16_t>> mLayerVoxelLU{}; //! helper structure to lookup a layer based on radius
```

Comment:
We must not have `std::` members in classes that go to GPU.
Also, we must not put an #ifdef here, since we must make sure that the size of the class is identical on GPU and host, otherwise copying the class will corrupt it.